player.gd

```diff
@@ -2,9 +2,12 @@ extends Area2D
 
 signal hit
 
+
+const MAX_SPEED = 400
+
 var speed = 0
-var acceleration = 15
-var coasting_deceleration = 150
+var acceleration = 900
+var coasting_deceleration = 400
```

Comment:
this looks like it would have a significant effect on player handling -- what were you hoping to change with this?

Reply:
I just tweaked values until I made them work more smoothly with the tilt formula, changed the acceleration formula to use delta to account for framerate best practices